src/main/java/codeu/controller/AjaxGetMessages.java

```diff
@@ -109,11 +111,19 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
             lastSize++;
         }
 
-        String messagesJson = new Gson().toJson(newMessageList);
-        System.out.println(messagesJson);
+        //Convert each message into an ajaxMessage and itt to ajaxMessageList
+        for(int i = 0; i < newMessageList.size(); ++i){
+          Message actualMessage = newMessageList.get(i);
+          AjaxMessage convertedMessage = AjaxMessage.toAjaxMessage(actualMessage);
+
+          ajaxMessageList.add(convertedMessage);
+        }
 
-        //Add name parameter instead of UUID to each message
+        //Convert ajaxMessages into a json
+        String messagesJson = new Gson().toJson(ajaxMessageList);
```

Comment:
What does line 123 do?

Reply:
It converts the `ajaxMessageList` into a `String` in `json format`, so that I can access the attributes of each message in the front-end without having any issue at all!

This would be an example of a simple json `myObj = { "name":"John", "age":30, "car":null };`.
Retrieved form here: https://www.w3schools.com/js/js_json_objects.asp